scala/integration/intellij-bazel/src/org/jetbrains/plugins/scala/bazel/BazelTestRunLineMarkerContributor.scala

```diff
@@ -0,0 +1,77 @@
+package org.jetbrains.plugins.scala.bazel
+
+import com.intellij.psi.impl.source.tree.LeafPsiElement
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.PsiTreeUtil
+import com.intellij.execution.PsiLocation
+
+import org.jetbrains.bazel.ui.gutters.BazelRunLineMarkerContributor
+import org.jetbrains.plugins.scala.lang.lexer.ScalaTokenTypes
+import org.jetbrains.plugins.scala.lang.psi.api.expr.{ScInfixExpr, ScMethodCall, ScReferenceExpression}
+import org.jetbrains.plugins.scala.lang.psi.api.statements.ScFunctionDefinition
+import org.jetbrains.plugins.scala.lang.psi.api.toplevel.typedef.{ScClass, ScTypeDefinition}
+
+import org.jetbrains.plugins.scala.extensions.ObjectExt
+import org.jetbrains.plugins.scala.testingSupport.test.scalatest.ScalaTestConfigurationProducer
+
+import java.util
+
+class BazelTestRunLineMarkerContributor extends BazelRunLineMarkerContributor {
```

Comment:
(minor)
I would add a scaladoc link to `org.jetbrains.plugins.scala.testingSupport.test.ui.ScalaTestRunLineMarkerProvider` here as a reminder of where is the rest scala test frameworks handling is made